vulture_os/toolkit/api_parser/csc_domainmanager/csc_domainmanager.py

```diff
@@ -157,7 +157,18 @@ def execute(self):
 
                 self.update_lock()
 
-                self.write_to_file([json.dumps(log) for log in logs['events']])
+                splitted_logs = []
+                for log in logs['events']:
+                    if 'records' in log.get('event'):
+                        for record_group in log['event']['records'].items():           
+                            for record in record_group[1]:
+                                splitted_log = log
```

Comment:
```suggestion
                                splitted_log = deepcopy(log)
```
May not be useful, but I prefer to make sure